ANSWERS.md

```diff
@@ -0,0 +1,23 @@
+1. Proptypes are used to check the form of data a component is 
+    given. This way, developers will be notified if they have
+    passed down the wrong data to a component and change it 
+    immediately instead of them spending time trying to find
+    what they did wrong
+
+2.  A life-cycle event is one of the events in a react component's 
+    lifecycle. They could be one of [mount, update, and unmount], 
+    developers can choose to do something when these events occur.
+
```

Comment:
Well, yes and no. React lifecycle methods each have a _very specific_ functionality. Developers call them when needed and generally don't alter the methods themselves.